frontend/src/components/ThemedButtons/ThemedButton.tsx

```diff
@@ -22,7 +26,7 @@ function ThemedButton({
 	ariaLabel,
 	className,
 	tooltip,
-	tooltipAnchor = 'top-center',
+	tooltipPosition = 'top-center',
```

Comment:
Think we want the default to be bottom-center now.

Reply:
Good spot!